docs/List.md

```diff
@@ -700,6 +702,3 @@ const PostList = () => (
 );
 ```
 {% endraw %}
-
-**Tip**: The `List` component `classes` can also be customized for all instances of the component with its global css name `RaList` as [describe here](https://marmelab.com/blog/2019/12/18/react-admin-3-1.html#theme-overrides)
```

Comment:
why did you remove this section?

Reply:
@fzaninotto Apologies, that was a mistake from a merge from a previous branch. Fixed now!